src/main/fc/fc_msp.c

```diff
@@ -929,6 +930,12 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU16(dst, compassConfig()->mag_declination / 10);
         break;
 #endif
+	case MSP_EXTRA_ESC_DATA:
+        for (int i = 0; i < getMotorCount(); i++) {			
```

Comment:
Please change this to send the motor count as the first byte. This way, it will be possible to extend the command at a later stage by adding more fields at the end, if necessary.

Reply:
Done